cmd/common-methods.go

```diff
@@ -576,4 +577,5 @@ type uploadSourceToTargetURLOpts struct {
 	multipartSize       string
 	multipartThreads    string
 	updateProgressTotal bool
+	preventStat         bool
```

Comment:
```suggestion
	skipStat         bool
```
I think `skipStat` is better name